src/default.js

```diff
@@ -76,9 +76,21 @@
     if (uaData) {
       try {
         payload.mobile = uaData.mobile;
-        var brand = uaData.brands.slice(-1)[0];
-        payload.browser = brand.brand;
-        payload.browserVersion = brand.version;
+        payload.brands = stringify(uaData.brands);
+
+        // Request platform information if this is available
+        if (isFunction(uaData.getHighEntropyValues)) {
+          uaData
+            .getHighEntropyValues(["platform", "platformVersion"])
+            .then(function (highEntropyValues) {
```

Comment:
This doesn't work, right? Because this promise could resolve after our page view request. We somehow need to wait for it before sending the page view.

Reply:
I pushed an update now that delays `sendPageView` if `getHighEntropyValues` is available and has not yet been requested once before. Please let me know if this is more like what you meant! :)